### PR TITLE
[nativeaot][tests] Update bundle zip path for NativeAOT sample app

### PR DESCRIPTION
**Pull request by kotlarmilos**
This PR updates bundle zip path for NativeAOT sample app by using a shared job between Mono and Native AOT measurements.

Follow-up https://github.com/dotnet/runtime/pull/85317

### CODE REVIEW

File: eng/pipelines/coreclr/templates/perf-job.yml
Comment (ivanpovazan) [inline eng/pipelines/coreclr/templates/perf-job.yml:319]: Might not be that relevant and I am not sure at which point these names are visible, but just wondering whether there should be a distinction between MonoAOT and NativeAOT in these names and paths, to avoid confusion.

Comment (kotlarmilos) [reply eng/pipelines/coreclr/templates/perf-job.yml:250]: Thanks for pointing out, the code has been simplified 🚀

**Comment by kotlarmilos** [comment]
This PR is ready for review. The performance results can be found at the dotnet-runtime-perf pipeline. Please note that there is a known issue https://github.com/dotnet/performance/issues/3054 with the Mono startup measurements, which is not related to the changes in the PR.

**Review by ivanpovazan — APPROVED** [review]
LGTM! 
However, I think someone from the infra team should approve it as well, before merging.

**Review by LoopedBard3 — APPROVED** [review]
This looks great! Looks good for merge presuming we have a test run of the pipeline with these changes 👍. I will likely also have to update the power-bi so will make a task to follow up on that.

**Pull request merged**
